ote/src/clj/ote/integration/import/gtfs.clj

```diff
@@ -301,27 +301,31 @@
 (defn download-and-store-transit-package
```

Comment:
This receives quite many parameters and I was wondering if this fn does more than it should

Reply:
Agree, but that's something for a new ticket :)